# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/cookiestore/PCookieStore.ipdl
# Commit: 675785853d7a
# Full Hash: 675785853d7a40123826e1b47c49d68f87bd4bd0
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2024-12-17 17:00:49
# Regressor Bug: 1915355
# File Overlap Count: 1
# Description:
#   Bug 1915355 - Part 5: Exempt third-party cookie blocking if the channel is on the exception list. r=cookie-reviewers,bvandersloot,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D223384
# ==============================================================================

diff -r de6a26d3e3df -r 675785853d7a dom/cookiestore/PCookieStore.ipdl
--- a/dom/cookiestore/PCookieStore.ipdl	Tue Dec 17 08:57:31 2024 +0000
+++ b/dom/cookiestore/PCookieStore.ipdl	Tue Dec 17 08:57:31 2024 +0000
@@ -36,6 +36,7 @@
                    bool thirdPartyContext,
                    bool partitionForeign,
                    bool usingStorageAccess,
+                   bool isOn3PCBExceptionList,
                    bool matchName,
                    nsString name,
                    nsCString path,
@@ -46,6 +47,7 @@
                    bool thirdPartyContext,
                    bool partitionForeign,
                    bool usingStorageAccess,
+                   bool isOn3PCBExceptionList,
                    nsString name,
                    nsString value,
                    bool session,
@@ -60,6 +62,7 @@
                       bool thirdPartyContext,
                       bool partitionForeign,
                       bool usingStorageAccess,
+                      bool isOn3PCBExceptionList,
                       nsString name,
                       nsString path,
                       bool partitioned,